tests/_fixtures/metronome/job.json

```diff
@@ -111,7 +111,7 @@
         }
       ]
     }
-  ] ,
```

Comment:
Ok.. I'll not change it, as the cypress versions (0.16.4) I'm using, seem to be is ok with it.